frontend/packages/integration-tests-cypress/tests/crud/k8-openshift-cruds.spec.ts

```diff
@@ -69,8 +69,7 @@ describe('Kubernetes resource CRUD operations', () => {
     describe(kind, () => {
       const name = `${testName}-${_.kebabCase(kind)}`;
 
-      const createResource = () => {
-        cy.log(`Create ${kind}: ${name}`);
+      it(`creates ${kind} named '${name}'`, () => {
```

Comment:
Don't add a generated name into the test. We want the title to be the same every run.

Reply:
ok, fixed, pls see updated output in PR Description